lib/index.js

```diff
@@ -107,7 +107,7 @@ var WebpackBrowserPlugin = function () {
         var url = options.url.replace('${port}', ':' + options.port);
         return url + '/' + this.options.publicPath;
       } else {
-        return options.url + ':' + options.port.toString() + '/' + this.options.publicPath;
+        return (options.port ? options.url + ':' + options.port.toString() : options.url) + '/' + this.options.publicPath;
```

Comment:
Why not 
`return options.url + (options.port ? ':' + options.port.toString() : '') + '/' + this.options.publicPath;`
?